AirCasting/ABConnector/SDCardAirBeamServices.swift

```diff
@@ -74,8 +73,8 @@ class BluetoothSDCardAirBeamServices: SDCardAirBeamServices {
                           self.finishSync { completion(.failure(SDCardSyncError.unexpectedMetadataFormat)) }
                           return
                       }
-                self.currentSessionTypeExpected = measurementsCount
-                progress(.metadata(SDCardMetaData(sessionType: self.currentSessionType!, measurementsCount: measurementsCount)))
+                currentSessionTypeExpected = measurementsCount
```

Comment:
couple topics to discuss:
- if we encounter unexpected format, we should rather 0 the counter (that's what we do in Android
- structure the code so it's more clear what step of sync this is
- we also get meta data once the card is cleared (SD_DELETE_FINISH) and I think we are not handling it here?